### ISSUE TITLE
Color Picker - Thumb doesn't have enough contrast

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#8917]
The thumb control border does not meet 3:1 contrast for non-text interactive elements.
![image](https://user-images.githubusercontent.com/20363037/57045369-37a8ad80-6c22-11e9-8c8a-2f3555584609.png)

**Comment by atneik** [comment]
@betrue-final-final what color of Gray should we be using to meet the contrast?

**Comment by atneik** [comment]
Ben asked to Gray110 - Gray130 colors depending upon what is supported by Theming. @Vitalius1 do you know which color will work throughout?

**Comment by atneik** [comment]
@betrue-final-final the thumb fill changes color as per the selected color. Even with a darker Gray how do we make sure the contrast is maintained no what color is chosen?    

Using Fluent styles:
![image](https://user-images.githubusercontent.com/785361/57342080-fddc1900-70f1-11e9-93c6-40009b8d61bb.png)

---

Using current default styles:
![image](https://user-images.githubusercontent.com/785361/57342585-64623680-70f4-11e9-8a76-31debed9bb0b.png)

I like how the current styles play with border color and shadow to make sure they work different colors.

**Comment by betrue-final-final** [comment]
Yeah, current styles works much better inside the colors. This really is a special case.

**Comment by atneik** [comment]
@betrue-final-final The fix for fabric-7 has been merged but wasn't sure how we handle in case of fluent-theme (which required a minor API change). So the fix isn't there in fluent-theme yet

**Comment by micahgodbolt** [comment]
This fix is live in the current 7,0 release, which seems to resolve the issue. Can we close this down @atneik. Is there work left to do here? Is there a concern with Fabric 6 using the Fluent theme?

**Comment by atneik** [comment]
@micahgodbolt I'd need @betrue-final-final to have the final say on the issue being addressed in fluent-theme. It wasn't clear how to go about fixing it stylistically in fluent-theme.

**Comment by micahgodbolt** [comment]
when you say 'fluent-theme' do you mean the fluent theme package? The package that we apply to Fabric 6 to make it look like Fabric 7? I don't understand what ben needs to answer (who is on vaca till next month)

**Comment by Vitalius1** [comment]
I doubt this issue is critical in the context of the `fluent-theme` package as it was meant for just a transitional process from anyone upgrading from 6 to 7 and needing some time to adjust the apps styles so that they play well with fluent fabric controls on the page. I consider if it's fixed on master (fabric 7) there is no need into investing anymore into `fluent-theme` parity. We already did a bunch of other fluent updates on master and never backported them to the theme package.

**Comment by atneik** [comment]
@micahgodbolt Yes, I meant fluent theme package.

@Vitalius1 That makes sense to me. Closing the issue.
